framework/auth/decorators.py

```diff
@@ -80,6 +83,21 @@ def wrapped(*args, **kwargs):
 
     return wrapped
 
+def ember_flag_is_active(flag_name):
```

Comment:
I don't think this belongs in `framework.auth`. How about moving it to `website.ember_osf_web.views` or `website.ember_osf_web.decorators`?

Reply:
good point, moved.